src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => (calculator.result += num),
+    subtract: (num) => (calculator.result -= num),
```

Comment:
[CODE STYLE] - Instead of `calculator.result -= num`, you can use `this.result -= num` to refer to the current object.